src/gui/editors/PianoRoll.cpp

```diff
@@ -152,6 +152,8 @@ PianoRoll::PianoRoll() :
 	m_zoomingModel(),
 	m_quantizeModel(),
 	m_noteLenModel(),
+	m_scaleModel(),
```

Comment:
Isn't the default CTOR called anyways if there is no CTOR specified? In other words, do these two lines have any functional impact?

Reply:
I added those lines for consistency, there were other model members in member initialization list. Yes, it isn't related with this bug, but I didn't create separate commit because I think changes are similar.